src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -62,8 +62,9 @@
                         <span class="relative items-center" {{ stimulus_controller('araise/core-bundle/dropdown') }}>
                             <button
                                 class="items-center justify-center hover:text-neutral-700"
-                                {{ stimulus_action('araise/core-bundle/dropdown', 'toggle') |
+                                {{ stimulus_action('araise/core-bundle/dropdown', 'close', 'dropdown:open@window') |
```

Comment:
Nit: Should we maybe prefix those events with something araise related so we lower the risk of collisions? e.g. `araise-dropdown:open`?

Might be something to consider for planned upcoming major updates; introducing prefixes for all controller names and events.